install/gcp/upgrade_tools/db_migrator.py

```diff
@@ -0,0 +1,71 @@
+import sys
+
+from sqlalchemy.exc import OperationalError
+
+import google.cloud.forseti.services.scanner.dao as scanner_dao
+import google.cloud.forseti.services.inventory.storage as inventory_dao
+import google.cloud.forseti.services.dao as general_dao
+
+from google.cloud.forseti.common.util import logger
+
+
+DEFAULT_DB_CONN_STR = 'mysql://root@127.0.0.1:3306/forseti_security'
+LOGGER = logger.get_logger(__name__)
+
+
+def migrate_schema(engine, base):
+    """Create all tables in the database if not existing.
+
+    Args:
+        engine (object): Database engine to operate on.
+        base (Base): Declarative base.
+    """
+    # Create tables if not exists.
+    base.metadata.create_all(engine)
+    base.metadata.bind = engine
+
+    # Update schema changes.
+    # Find all the child classes inherited from declarative base class.
+    base_subclasses = _find_subclasses(base)
+
+    # Find all the Table objects for each of the classes.
+    # The format of tables is: {table_name: Table object}.
+    tables = base.metadata.tables
+
+    schema_update_method_name = 'schema_update'
+
+    for subclass in base_subclasses:
+        schema_update = getattr(subclass, schema_update_method_name, None)
+        if callable(schema_update) and subclass.__tablename__ in tables:
+            LOGGER.info('Updating table %s', subclass.__tablename__)
+            # schema_update will require the Table object.
+            try:
+                schema_update(tables.get(subclass.__tablename__))
+            except OperationalError:
+                LOGGER.info('Failed to update db schema, table=%s',
```

Comment:
This should be LOGGER.error(), rather than info()

If this is `OperationalError`, can we be more specific in the log message what might have happened?

Reply:
This is expected when the table already has that column, I think info makes more sense so user won't need to worry about seeing the message at error level. Added log message in the update_schema method so we know what is failing.

Reply:
There is no log message in the `update_schema()`

Reply:
There is an info log in the update_schema method - LOGGER.info('Attempting to create column: %s', col_name)